packages/compass-connections/src/stores/connections-store.ts

```diff
@@ -370,7 +374,8 @@ export function useConnections({
     ) => {
       try {
         dispatch({ type: 'set-active-connection', connectionInfo });
-        onConnected(connectionInfo, dataService);
+        dataService.addReauthenticationHandler(reauthenticationHandler.current);
```

Comment:
If creating and managing data services is now a job of connection manager, can we move this reauth handler adding to connection manager? Seems like something we just need to pass to manager constructor as an option

Reply:
Yea that sounds good, I will take care of this

Reply:
Also addressed in my recent update.